Include <glib/gprintf.h> for the g_printf function.  This avoids an
implicit function declaration and build failures with future
compilers.

diff --git a/test/test-async-directory.c b/test/test-async-directory.c
index 2698d8e6e41f9ead..8d7e0f320edbb5fc 100644
--- a/test/test-async-directory.c
+++ b/test/test-async-directory.c
@@ -24,6 +24,7 @@
 #include <config.h>
 
 #include <glib.h>
+#include <glib/gprintf.h>
 #include <libgnomevfs/gnome-vfs-async-ops.h>
 #include <libgnomevfs/gnome-vfs-init.h>
 #include <stdio.h>
